### PR TITLE
Update the design of the success screen

### PR DESCRIPTION
**Pull request by ShcherbaDev**
Resolve for https://phabricator.wikimedia.org/T241447

### CODE REVIEW

**Comment by tonythomas01** [comment]
Great first step. Thank you. Can we have "The following files were uploaded!" instead of "Your selected file(s):" on the upload finish screen ?

**Comment by tonythomas01** [comment]
One more thing to think about (out of GCI) is that whether we should have a small thumbnail preview of the uploaded icon also.

**Comment by ShcherbaDev** [comment]
> Great first step. Thank you. Can we have "The following files were uploaded!" instead of "Your selected file(s):" on the upload finish screen ?

Yep. I just copied the text from the previous stem. Nothing was written about that on task description so I decided to do like that.

> One more thing to think about (out of GCI) is that whether we should have a small thumbnail preview of the uploaded icon also.

Is it should be instead of the "URL" column?

**Comment by tonythomas01** [comment]
> Is it should be instead of the "URL" column?

Well, the thing is we might actually want to keep the URL column. How about something like this: 
Thumbnail | Name of file | Link to upload

**Comment by ShcherbaDev** [comment]
Got it. I'll work on this later

File: uploader/templates/upload.html [hunk 1]
```diff
@@ -1,383 +1,408 @@
...
+      },
+      success: function(data) {
+        $(".step-3").hide();
+        $(".step-1-2").hide();
+        $(".step-4").show();
+        $(".results-panel").append("<tbody>");
+        data.forEach(function(item) {
+          $(".results-panel").append(
+            "<tr><td><img src='" +
+              generateThumbNailLink(item.url) +
```
Comment (tonythomas01) [inline uploader/templates/upload.html]: Sadly, we cannot use this at this point. item.url is a link to the wiki file. We should simply use the same link and probably add some width and height stylings to it.

**Pull request merged**